app/src/gecko/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -274,6 +277,14 @@ public boolean onLoadUri(GeckoSession session, String uri, GeckoSession.Navigati
                         return true;
                     }
 
+                    // Check if we should handle an external link
+                    final Uri urlToURI = Uri.parse(uri);
```

Comment:
`urlToURI` is a bit hard to read. However I can't come up with a better name. Feel free to just merge if you have no idea either. :)